website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Make Public also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Workspace</b></td>
+          <td>View, Execute</td>
+          <td>Allows making all applications publicly accessible within the given workspace, and allows viewing and running queries to view the application when applications are publicly accessible.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Application</td>
+          <td>View, Execute</td>
+          <td>Allows making the given application publicly accessible, and allows viewing and running queries to view the application when it's publicly accessible.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Export permission
+
+The **Export** permission permits users to export data and resources from Appsmith. When the **Export** permission is assigned to a workspace or an application, it, along with the automatically assigned permissions, are inherited by the child resources.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Application Resources" value="execute-application-resources">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Export also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Workspace</b></td>
+          <td>View, Execute</td>
+          <td>Allows viewing application, exporting data and resources, and running queries required to view application and exporting data within the given workspace.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Application</td>
+          <td>View, Execute</td>
+          <td>Allows viewing application, exporting data and resources, and running queries required to view application and exporting data within the given application.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Invite User permission
+
+The Invite User permission it permits users to invite other users to Appsmith workspace or applications and provides access to the resources within workspace.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Groups & Roles" value="invite-user-groups-and-roles">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Invite User also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Groups</b></td>
+          <td>View</td>
+          <td>Allows viewing the groups, and inviting users to workspaces and applications within the instance.</td>
```

Comment:
This isn't true. Invite User allows you to invite a user to the group only.

Reply:
rewrote the description